RELEASE.md

```diff
@@ -0,0 +1,35 @@
+Release Instructions
+--------------------
+
+Pre-Commit
+----------
+1. pushd ngrok-java-native && fix-n-fmt && popd
+1. export NGROK_AUTHTOKEN="<your_authtoken>"; make all
+
+Prepare Release
+---------------
+NOTE: this step needs to make commits to main, which will be blocked by branch controls. You will
+need to get in contact with an admin to temporarily remove the branch control while completing this step.
```

Comment:
hm, maybe I'm not seeing it — is there a reason why the commits can't be a PR to main as part of the workflow?

Reply:
In it's current state, the "Prepare Release" workflow creates a new tag with the version. I think that's probably the main point of contention. If you do it on a branch, I don't think the tagging would work properly.

Now, there's nothing that says the prerelease HAS to create the new tag, but that's the way it currently is. I know Niki mentioned that this quirk is a known issue, it just hasn't been intrusive or bothersome enough for anybody to invest the time to fix it.

(On the note about fixing it, I'm curious how one would go about testing drastic changes to the workflow without inadvertently creating new version, but I digress)

Reply:
ok, thanks for the context Ryan. 

>  If you do it on a branch, I don't think the tagging would work properly.

yeah, I'm definitely not the expert here. I'm ok with merging this in as-is and working with the quirk and improving in the future if we need.

Reply:
sounds good!